python/orca/src/bigdl/orca/learn/pytorch/pytorch_ray_estimator.py

```diff
@@ -209,6 +211,13 @@ def fit(self,  # type:ignore[override]
             callbacks=callbacks,
         )
 
+        if self.backend == "ray":
+            self.setup(params=self.setup_params,
```

Comment:
setup in Line145 will already create Ray TorchRunners right? Here you call setup again, will create new TorchRunners?

Reply:
Yes, it will, only fit will create new runners.

Reply:
I mean there are already runners at init time of the Estimator right? Fit will create new runners?

Reply:
yes, that's what I mean😂, init estimator will create runners. When calling fit, it will also create new runners.

Reply:
I don't think it is a good idea to create new runners? Better just use the runners when init? BTW, the runners in init will already occupy all the ray cores, how can you create new runners?